### ISSUE TITLE
UI for commit splitting a la GitUp

### ISSUE THREAD

**Issue by nickolay** [sublimehq/sublime_merge#104]
**Problem description**

As you develop something new and non-trivial, you may commit to checkpoint your work.

If you want to publish a clean, easy-to-review list of changes that you settled on, instead of the way you settled on the eventual solution, you usually rebase your changes.

One common action during rebase is [splitting a commit](https://stackoverflow.com/questions/4307095/how-can-i-split-up-a-git-commit-buried-in-history) into two.

**Preferred solution**

GitUp has a very nice UI that lets one split a commit into two: https://github.com/git-up/GitUp/wiki/Splitting-Commits-with-GitUp - it would be great if this was duplicated in sublime at some point.

In short, it lets you select a commit and presents it in a split view, similar to staging UI, but instead of working directory / staging area you move files/hunks between an "Old commit" and a "New commit".

**Comment by jskinner** [comment]
Sublime Merge is now able to split commits, via 'Edit Commit/Edit Commit Contents' as of Dev Build 1085 at https://www.sublimemerge.com/dev
